cpp/tests/join/join_tests.cpp

```diff
@@ -44,6 +44,28 @@ constexpr cudf::size_type NoneValue =
   std::numeric_limits<cudf::size_type>::min();  // TODO: how to test if this isn't public?
 
 struct JoinTest : public cudf::test::BaseFixture {
+  void check_gather_maps(
```

Comment:
This function is going to be called multiple times in this file. Whenever a test failed, you can't know at which line it failed---you always see "failure at line 67", because you only have the check `CUDF_TEST_EXPECT_TABLES_EQUIVALENT(*sorted_gold, *sorted_result);` at line 67.

I would recommend to have this function returns a pair of `cudf::table`, then call `CUDF_TEST_EXPECT_TABLES_EQUIVALENT(*sorted_gold, *sorted_result);` at each of the call sites.

Reply:
Thanks, this will make it easier to debug the specific test case if it fails.